### DISCUSSION TITLE
CodeSpaces on iPad

### DISCUSSION THREAD

**Discussion by horacioh** [community/community#26336] [Codespaces]
<p>Hello folks :wave:t3:</p>
<p>I’ve been using CodeSpaces on my iPad for note taking and I gotta say the experience is far better that what I was expecting!! Kudos to the team involved!!</p>
<p>However, there where some issues I found using it and I want to give proper feedback about it.</p>
<p>Q: What would be the best way to file the issues I found? Should I just open a topic here for each issue or what would be the best way to suit the team’s workflow?</p>
<p>Thanks in advance!!</p>

**Comment by 2percentsilk** [comment, marked as answer]
<p>You can feel free to share those issues here and the team can make sure they’re appropriately filed on our backlog.</p>

**Comment by horacioh** [comment]
<p><strong>Awesome!</strong></p>
<p>Thanks!</p>
